js/widgets/forms/clearButton.js

```diff
@@ -47,7 +47,7 @@ define( [
 		},
 
 		_addClearBtn: function() {
-
+			this.widget().children('input').addClass("ui-hide-default-clear") ; /*Removes default clear button if jqm clear button is ON*/
```

Comment:
@kapilgarg1996 
- please use double quotes for strings
- please put single spaces around strings, i.e., ```( "abc" )```
- please place the comment on the previous line, leaving a blank line before it, and use ```//``` as the comment delimiter